pulsar-broker/src/test/java/org/apache/pulsar/client/impl/ServiceUrlQuarantineTest.java

```diff
@@ -82,16 +83,16 @@ protected void setup() throws Exception {
                                 .serviceUrl(binaryServiceUrlWithUnavailableNodes)
                                 .serviceUrlQuarantineInitDuration(0, TimeUnit.MILLISECONDS)
                                 .serviceUrlQuarantineMaxDuration(0, TimeUnit.MILLISECONDS)
-                                .operationTimeout(TIMEOUT_SECONDS, TimeUnit.SECONDS)
-                                .lookupTimeout(TIMEOUT_SECONDS, TimeUnit.SECONDS)
+                                .operationTimeout(TIMEOUT_MS, TimeUnit.SECONDS)
+                                .lookupTimeout(TIMEOUT_MS, TimeUnit.SECONDS)
                                 .build();
         this.pulsarClientWithHttpServiceUrlDisableQuarantine =
                 (PulsarClientImpl) PulsarClient.builder()
                         .serviceUrl(httpServiceUrlWithUnavailableNodes)
                         .serviceUrlQuarantineInitDuration(0, TimeUnit.MILLISECONDS)
                         .serviceUrlQuarantineMaxDuration(0, TimeUnit.MILLISECONDS)
-                        .operationTimeout(TIMEOUT_SECONDS, TimeUnit.SECONDS)
-                        .lookupTimeout(TIMEOUT_SECONDS, TimeUnit.SECONDS)
+                        .operationTimeout(TIMEOUT_MS, TimeUnit.SECONDS)
+                        .lookupTimeout(TIMEOUT_MS, TimeUnit.SECONDS)
```

Comment:
should use `TimeUnit.MILLISECONDS`?

Reply:
> Yes, Pulsar CI is broken currently and would be fixed by #24569.
> 
> In the meantime, it's sufficient to make the changes that I pointed out in my review (please address [#24574 (comment)](https://github.com/apache/pulsar/pull/24574#issuecomment-3126811529)). Just validate the test locally and push the changes to this PR so that we can proceed in addressing this PR while Pulsar CI is blocked.

Hi lari, I have passing tests locally, and the CI in the forked repo has no `ServiceUrlQuarantineTest` related failures.